docs/content/Getting-Started/user-quickstart-test.md

```diff
@@ -173,12 +173,12 @@ echo The mailbox for ks-edge-cluster1 is $MB1
 MB2=$(KUBECONFIG=$SM_CONFIG kubectl get spaces -n spaceprovider-default -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "ks-edge-cluster2") | .name')
 echo The mailbox for ks-edge-cluster2 is $MB2
 
-MB1_SPACE="${PWD}/temp-space-config/${MB1}"
-kubectl-kubestellar-get-config-for-space --space-name $MB1 --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT --output $MB1_SPACE
-MB2_SPACE="${PWD}/temp-space-config/${MB2}"
-kubectl-kubestellar-get-config-for-space --space-name $MB2 --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT --output $MB2_SPACE
-WMW1_SPACE_CONFIG="${PWD}/temp-space-config/spaceprovider-default-wmw1"
-kubectl-kubestellar-get-config-for-space --space-name wmw1 --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT --output $WMW1_SPACE_CONFIG
+MB1_KUBECONFIG="${PWD}/${MB1}.kubeconfig"
```

Comment:
This is a general comment that I'll put here.  These generated files are not always cleaned up.  Sometimes we run these tests on our own machine and then have to clean things up manually.  If the generated files go into a subdirectory the cleanup is easier.  Can we decide on a convention where the stored kubeconfig files are in a sub dir?

Reply:
Or in tmp.

Reply:
Good point.

Reply:
We may categorize them as follows:
- user (e.g., example1 or a quickstart script) chose where to put the file;
- script chose where to put the file.

There is the potential for conflicts if both categories go to the same directory, because each author does not know what the other is doing.

In the latter case my intent has been that the script deletes the file when done, except for the cases where there is an on-going need for the file (konnector, example-backend).

Reply:
I think I will change the example1 and quickstarts to locate those kubeconfigs in a directory that is cleared at the _start_.

Reply:
Excellent decision.  Thank you.

Reply:
Done in the push to 2781ec836

Reply:
Sounds good